.github/workflows/ci.yaml

```diff
@@ -11,28 +11,17 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
-
-      - name: Setup PATH
-        shell: bash
-        run: |
-          mkdir -p "$HOME/.local/bin"
-          echo "$HOME/.local/bin" >> $GITHUB_PATH
+      - uses: actions/checkout@v4
 
       - name: Install jsonnet
-        shell: bash
         run: |
-          wget https://github.com/google/go-jsonnet/releases/download/v${JSONNET_VERSION}/go-jsonnet_${JSONNET_VERSION}_Linux_x86_64.tar.gz
-          tar --extract --file=go-jsonnet_0.20.0_Linux_x86_64.tar.gz jsonnet
-          mv jsonnet "$HOME/.local/bin"
-          chmod u+x "$HOME/.local/bin/jsonnet"
```

Comment:
By piping into tar, we don't create a file to clutter the working dir. Whats extracted are multiple files as well, not just jsonnet - also jsonnet-lint for example, and they are packaged with chmod +x already in the tar so they extract like that as well.